en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+            
+    Click the **+** icon below the `quantity` variable statement, and then click **API Calls**. Then click **Google Sheets**.
+    
+    To access the specific account you need to use for this scenario, click **Connect to Google Sheets**, and then click **Proceed**. Click on the Google Account of your choice and click **Allow** to allow Choreo to access it as described on the page. Then click **Save**.
+    
+    Now you can configure the Google sheets Google account you selected to receive API calls. 
+    
+    To do this, click the **+** icon below the Google Sheets connection you just added. Click **API Calls**, and then click **sheetsEndpoint** under **Choose existing connection**.
+    
+    In the **Operation** field, select **Get values from a column**. Then enter the following information in the rest of the fields that appear:
+    
+    | **Field**                  | **Value**                                                                                                       |
+    |----------------------------|-----------------------------------------------------------------------------------------------------------------|
+    | **SpreadsheetId**          | The ID of your Google sheet. This can be derived from the URL of your Google sheet. e.g., If the Google Sheet URL is `https://docs.google.com/spreadsheets/d/1wHOdJgJzN6V9-vBQVSLEYxpYDp5XnpIcNqgj-h9pX4Q/edit#gid=0`, the google sheet ID is `1wHOdJgJzN6V9-vBQVSLEYxpYDp5XnpIcNqgj-h9pX4Q`. |                                                                                  |  
+    | **Worksheet Name**         | The name of the sheet with the inventory records.                                                               |
+    | **Column**                 | The ID of the column in which you have added values for the `item` variable (i.e., `A` in the given example image). |
+    | **Response Variable Name** | `itemIDs`                                                                                                       |
+        
+    Here, you are adding a connection that retrieves the number of items currently available on column `A`, `Sheet1` of the `inventory` Google Sheet.
+        
+    Click **Save**.
+        
+ 6. For the service to cast the information is retrieved from the Google sheet as a response to the order, add a variable.
+ 
+    To do this, click the **+** icon below the Google Sheet connection, and then click **Variable**.
+    
+    Enter the following information:
+    
+    | **Field**         | **Value**        |
+    |-------------------|------------------|
+    | **Type**          | **other**        |     
+    | **Other Type**    | `string[]`       |
+    | **Name**          | `respondMessages`|
+    | **Expression**    | `[]`             |
+        
+    Click **Save**.
+        
+ 7. Once the response is received, the service needs to log it.
```

Comment:
Instead of saying `needs to log it`, can we reword it to say we better log it to ... etc as It is not a must to log something.